build/index.js

```diff
@@ -236,11 +236,9 @@ async function buildDocument(document, documentOptions = {}) {
         // message.
         error.updateFileInfo(document.fileInfo);
         throw new Error(
-          `MacroInvocationError trying to parse ${error.filepath}, line ${error.line} column ${error.column} (${error.error.message})`
+          `${error.filepath}:${error.line}:${error.column}: error: MacroInvocationError - ${error.error.message}`
```

Comment:
I think this is a bit odd, that identifier comes before the "name". 
```
TypeError: unable to parse foo.js:123
```
...makes sense and something I'd expect to see and easily parse in my head. 
But...
```
foo.js:123 TypeError - unable to parse
```

Reply:
I think the `error:` in mine can be dropped, because the assumption is an error if it's not set. Good to keep if later you want to split warnings vs. errors in this format though

Reply:
In terms of the "head parsing", compare this to what you see when you run ESLint in your terminal. The alternate format that's supported is a heading for the file, then `line:column: Message`, but then you can't directly click to the position from the terminal

Reply:
But if you make it 
```
MacroInvocationError: ${error.error.message} ${error.filepath}:${error.line}:${error.column}
```

Reply:
I think the reason the tools don't do that format is that the regex to match it gets uglier with an arbitrary string(s) in front, instead of known format in front, then variable strings afterwards

Reply:
All of that aside, what's the original intention/goal with this? Is it to be able to click the filepath as it's displayed in the terminal when the error happens?

Reply:
It's a tie, with that, and being able to add a problem matcher in `mdn/content` that will leave a comment on the particular line in a PR

Reply:
In that case, and I trust that you know what you're doing, then I won't block due to not fully understanding it. 

@escattone Can you take a look too?